crates/astria-celestia-client/src/client.rs

```diff
@@ -146,12 +146,12 @@ pub trait CelestiaClientExt: BlobClient {
     }
 
     /// Submits sequencer `blocks` to celestia after converting and signing them, returning the
-    /// height at which they were included.
+    /// height at which they were included. Sequencer data for each block will be posted to a
```

Comment:
For doc comments try to have a short summary on the first line followed by a space, and then more text like:
```rust
/// Summary of what the function is supposed to do
///
/// More information about the function, interesting background stories,
/// where the function has been, early life, career progression, family related
/// trivia.
```
(the original title was already too long actually)